src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -163,6 +168,10 @@ function createFirstChild(returnFiber, existingChild, newChildren, priority) {
 
 // TODO: This API won't work because we'll need to transfer the side-effects of
 // unmounting children to the returnFiber.
-exports.reconcileChildFibers = function(returnFiber : Fiber, currentFirstChild : ?Fiber, newChildren : ReactNodeList, priority : PriorityLevel) : ?Fiber {
+exports.reconcileChildFibers = function(
+  returnFiber : Fiber, 
+  currentFirstChild : ?Fiber, 
+  newChildren : ReactNodeList, 
+  priority : PriorityLevel) : ?Fiber {
```

Comment:
@zpao @sebmarkbage do we want to use `returnFiber : Fiber` or `returnFiber: Fiber` (space before the colon)? 

Also the closing `)` should be on a new line IMO.